ui/DesignSystem/Component/PeerSelector/Entry.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="typescript">
+  export let active: boolean = true;
```

Comment:
```suggestion
  export let disabled: boolean = true;
```

`active` seems a bit misleading as it refers more to something being selected or not.